src/Button/Button.cpp

```diff
@@ -88,7 +88,8 @@ boolean Button::update(){
 	} else if(_new_button_state){
 		_button_held();
 		return false;	// State hasn't changed
```

Comment:
Could you move this line to where you put return -1 and I'll merge

Reply:
done

Reply:
did already. Can you see my first coment. There was a updated Button.cpp